controllers/widget.js

```diff
@@ -153,6 +153,25 @@ function init(opts) {
     }
   }
 
+	_.each(opts.children, function (child) {
+
+		// fix: https://jira.appcelerator.org/browse/TC-3583
+		if (!child) {
+			return;
+		}
+
+		console.debug('adding footer/header views from xml', child.role);
```

Comment:
My case
```xml
<Widget id="form" src="nl.fokkezb.form" config="forms/project_add">
    <View class="headerView" role="headerView" formFactor="tablet">
        <Label id="titleLabel"/>
    </View>
    <Button/>
    <View class="footerView" role="footerView">
        <Button id="sendButton" onClick="sendButtonClicked"/>
    </View>
</Widget>
```

child of zero come with null, maybe TC-3583 fixed on platform but not with formfactor